src/browser.ts

```diff
@@ -163,10 +163,13 @@ export class VSBrowser {
      * @returns Promise resolving when all selected resources are opened and the workbench reloads
      */
     async openResources(...paths: string[]): Promise<void> {
+        if (paths.length === 0) {
```

Comment:
I have noticed that this will never possible happen as running the code with -r option without actually passing any argument results in:
```
> npm run compile && extest setup-and-run './out/src/test/**/resource*test.js' -u -s ~/test -r


> test-project@0.0.1 compile
> rimraf out && tsc -p ./

error: option '-r --open_resource <resources...>' argument missing
```

Reply:
Ah, nevermind, now I understand, you need a way to avoid actually opening anything if the flag was not used.